src/Razor/src/Microsoft.VisualStudio.RazorExtension/Properties/launchSettings.json

```diff
@@ -14,7 +14,7 @@
         "CPS_DiagnosticRuntime": "1",
         "CPS_MetricsCollection": "1",
         "ServiceHubTraceLevel": "All",
-        "SERVICEHUBDEBUGHOSTONSTARTUP": "ServiceHub.RoslynCodeAnalysisService.exe"
+        "SERVICEHUBDEBUGHOSTONSTARTUP": "ServiceHub.RoslynCodeAnalysisServiceS.exe"
```

Comment:
Curious, did Roslyn change the naming convention since I added this? Or did I just get it wrong?

Reply:
Ah no, the `S` stands for server GC. Servicehub uses a different process name depending on which mode you have set to use. The server GC tends to work a little better for my 128GB of ram-based-stupidity. I'll change this back as its the less common option.